exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -424,10 +418,11 @@ private ServerConnector createHttpsConnector(int port) throws Exception {
    * @return Initialized {@link ServerConnector} instance for HTTP connections.
    * @throws Exception
    */
-  private ServerConnector createHttpConnector(int port) throws Exception {
+  private ServerConnector createHttpConnector(int port, int acceptors, int selectors) throws Exception {
     logger.info("Setting up HTTP connector for web server");
     final HttpConfiguration httpConfig = new HttpConfiguration();
-    final ServerConnector httpConnector = new ServerConnector(embeddedJetty, new HttpConnectionFactory(httpConfig));
+    final ServerConnector httpConnector =
+        new ServerConnector(embeddedJetty, null, null, null, acceptors, selectors, new HttpConnectionFactory(httpConfig));
```

Comment:
Why not
 
```
new ServerConnector(embeddedJetty, acceptors, selectors, new HttpConnectionFactory(httpConfig));
```

Reply:
I don't think there is such constructor (only one that takes `SslContextFactory` as the last argument), please double check.